api.md

```diff
@@ -2654,65 +2656,232 @@ SELECT * FROM timescaledb_information.data_node;
+| `num_chunks` | Number of chunks |
+| `compression_enabled` |Is compression enabled on the hypertable?|
+| `is_distributed` | Is the hypertable distributed?|
+| `replication_factor` | Replication factor for a distributed hypertable|
+| `data_nodes` | Nodes on which hypertable is distributed|
+| `tablespaces` |Tablespaces attached to the hypertable |
 
-#### Sample Usage [](timescaledb_information-hypertable-examples)
+#### Sample Usage [](timescaledb_information-hypertables-examples)
 
-Get information about all hypertables.
+Get information about a hypertable.
 
 ```sql
-SELECT * FROM timescaledb_information.hypertable;
+CREATE TABLE dist_table(time timestamptz, device int, temp float);
+SELECT create_distributed_hypertable('dist_table', 'time', 'device', replication_factor => 2);
+
+SELECT * FROM timescaledb_information.hypertables
+  WHERE table_name = 'dist_table';
+
+-[ RECORD 1 ]-------+-----------
+table_schema        | public
+table_name          | dist_table
+owner               | postgres 
+num_dimensions      | 2
+num_chunks          | 3
+compression_enabled | f
+is_distributed      | t
+replication_factor  | 2
+data_nodes          | {node_1, node_2}
+tablespaces         | 
 
- table_schema | table_name | table_owner | num_dimensions | num_chunks | table_size | index_size | toast_size | total_size | distributed
---------------+------------+-------------+----------------+------------+------------+------------+------------+------------+--------------
- public       | metrics    | postgres    |              1 |          5 | 99 MB      | 96 MB      |            | 195 MB     | t
- public       | devices    | postgres    |              1 |          1 | 8192 bytes | 16 kB      |            | 24 kB      | f
-(2 rows)
 ```
 
-Check whether a table is a hypertable.
+## timescaledb_information.dimensions [](timescaledb_information-dimensions)
 
-```sql
-SELECT * FROM timescaledb_information.hypertable
-WHERE table_schema='public' AND table_name='metrics';
+Get metadata about the dimensions of hypertables, returning one row of metadata 
+for each dimension of a hypertable.  For a time-and-space-partitioned 
+hypertable, for example, two rows of metadata will be returned for the 
+hypertable.
 
- table_schema | table_name | table_owner | num_dimensions | num_chunks | table_size | index_size | toast_size | total_size | distributed
---------------+------------+-------------+----------------+------------+------------+------------+------------+------------+--------------
- public       | metrics    | postgres    |              1 |          5 | 99 MB      | 96 MB      |            | 195 MB     | t
-(1 row)
-```
+A time-based dimension column has either an integer datatype (e.g bigint, integer)
+or a time related datatype (e.g. timestamptz, timestamp, date).
+The `time_interval` column is defined for hypertables that use time datatypes 
+like timestamptz, timestamp, date. 
+Alternatively, for hypertables that use integer datatypes (e.g., bigint, 
```

Comment:
probably makes sense to list all datatypes here since there are only 3 smallint,integer,bigint